plugins/groq/package.json

```
@@ -24,16 +24,17 @@
   "author": "TheFireCo",
   "license": "Apache-2.0",
   "dependencies": {
-    "groq-sdk": "^0.3.3",
-    "zod": "^3.22.4"
+    "groq-sdk": "^0.3.3"
   },
   "peerDependencies": {
-    "@genkit-ai/ai": "^0.5.0",
-    "@genkit-ai/core": "^0.5.0"
+    "genkit": "^0.9.0-dev.2"
   },
   "devDependencies": {
+    "@types/hast": "^3.0.4",
```

```suggestion
    "@types/hast": "^3.0.4",
```
What do we use these types for?

---

I had an error in tsconfig.json without them